src/components/ui/icons/ChatCaret/index.tsx

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import Svg, { SvgProps } from '../Svg';
+
+const ChatCaret: React.SFC<SvgProps> = props => (
+  <Svg {...props} viewBox="0 0 4 9">
```

Comment:
Why doesn't this icon have the same viewBox dimensions as the others? What is different about the `ChatCaret` compared to the existing `Caret`?

Reply:
It's not really the same at all. Per the design, some chat bubbles will have the little fin on the bubble and some won't. It's not used in the same way as any other icon and would only be attached to a chat bubble  <div>.

Reply:
Maybe since this icon is so different and specific, it should only be defined in the html of the chat message component. What do you think?

Reply:
Thanks for explaining. I was having a hard time visualizing what this is used for / looked like. I can't think of any other scenario where this might be used, which makes it a good candidate for keeping it with the main chat message component. I'd recommend doing that, we can always isolate it if there ends up being a need for it somewhere else.

Reply:
<img width="243" alt="Screen Shot 2020-09-10 at 1 53 48 PM" src="https://user-images.githubusercontent.com/63013154/92804422-2cfd3d00-f36d-11ea-8f89-f6667a72ca31.png">

It's only the tiny tail on this chat bubble. I'm updating to just be part of the chat message component now.